main.go

```diff
@@ -90,6 +103,15 @@ It can be used in a curl-like fashion when benchmarking features are disabled.
 		return opts, errExit
 	}
 
+	// Read defaults from ~/.yab.ini
+	if user, err := user.Current(); err != nil {
```

Comment:
instead of using the user.Homedir, what do you think of using `$HOME`? `user.Current` relies on ldap / glibc which we might not need otherwise, and `$HOME` might be easier to add a simple unit test for